elixir/apps/domain/test/domain/actors_test.exs

```diff
@@ -3080,7 +2982,7 @@ defmodule Domain.ActorsTest do
       assert actor.deleted_at
 
       group = Repo.preload(group, :memberships, force: true)
-      assert group.memberships == []
+      assert actor.id not in Enum.map(group.memberships, & &1.actor_id)
```

Comment:
I believe this test had a bug. When `delete_actor` is called a few lines above, it deletes identities, which in turn calls `update_managed_group_memberships/1`. That will insert a membership for the actor created in this test on the first line.

So the end result - since we had two actors in this test and deleted one - is that the group should contain one membership with the actor created in this test.